packages/icons/svg/share.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' width='24px' height='24px'>
+    <title>Share Link</title>
+    <path
+      fill='#0068ef'
```

Comment:
this should be `currentcolor` not the primary blue color right?

Reply:
I believe the component wrapper passes that in and it overrides the svg's fill. It's confirmed working with the DS color props - I tested with creating some sample stories.